project_legal_management/views/lm_resolution_views.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2021 NuoBiT Solutions - Kilian Niubo <kniubo@nuobit.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl) -->
+<odoo>
+    <record id="lm_resolution_form_view" model="ir.ui.view">
+        <field name="name">lm.resolution.form.view</field>
```

Comment:
Same as before, check OCA guidelines https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#252security-view-and-action